frontend/packages/knative-plugin/integration-tests/features/serverless/domain-mapping.feature

```diff
@@ -0,0 +1,64 @@
+@knative
+Feature: Custom Domain Mapping Support
+              As a user, I would like to have the ability to provide a custom domain for the routes configured for serverless applications.
+
+
+        Background:
+            Given user has installed OpenShift Serverless Operator
+              And user has created Knative Eventing CR
```

Comment:
why only talking about `Knative Eventing` CR and not `Knative Serving` CR

Reply:
Added that as well.